.github/workflows/clang-c++26.yml

```
@@ -0,0 +1,185 @@
+  build-clang-cpp26-osx-stl:
+    name: Clang C++26 OSX - STL
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-26]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Build
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:6:9: Quote this to prevent word splitting

(shellcheck)